packages/@aws-cdk/toolkit/lib/api/io/private/codes.ts

```diff
@@ -1,20 +1,6 @@
 import { IoMessageCode } from '../io-message';
 
 export const CODES = {
-  // Default codes -- all 0000 codes
-  CDK_TOOLKIT_I0000: 'Default toolkit info code',
-  CDK_TOOLKIT_E0000: 'Default toolkit error code',
-  CDK_TOOLKIT_W0000: 'Default toolkit warning code',
-  CDK_SDK_I0000: 'Default sdk info code',
-  CDK_SDK_E0000: 'Default sdk error code',
-  CDK_SDK_WOOOO: 'Default sdk warning code',
-  CDK_ASSETS_I0000: 'Default assets info code',
-  CDK_ASSETS_E0000: 'Default assets error code',
-  CDK_ASSETS_W0000: 'Default assets warning code',
-  CDK_ASSEMBLY_I0000: 'Default assembly info code',
-  CDK_ASSEMBLY_E0000: 'Default assembly error code',
-  CDK_ASSEMBLY_W0000: 'Default assembly warning code',
```

Comment:
I don't want to allow explicitly using this code for now. When generating the docs page we can probably inject the description differently.

Reply:
I thought this would break the place where we generate `0000` codes

Reply:
It does not apparently!